docs/source/api/to_extensions_id.rst

```diff
@@ -0,0 +1,75 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-to_extensions-id:
+
+************************
+``to_extensions/{{ID}}``
+************************
+
```

Comment:
Since this is a new endpoint in 1.5, it should use `.. versionadded` to indicate that

Reply:
Doneso!